### PR TITLE
Vydasari/search

### PR DESCRIPTION
**Pull request by VyshnaviDasari**

### CODE REVIEW

File: src/operations-tracker-header/operations-tracker-header.tsx [hunk 1]
```diff
@@ -54,14 +50,14 @@ export const OperationsTrackerHeader = React.memo(
...
         </div>
         {openDescription && (
           <div className={classes.description}>
-            It monitors changes in cache, fired mutations and
-            activated/deactivated queries.
+            <span>
+                It monitors changes in cache, fired mutations and
+                 activated/deactivated queries.
+            </span>
+            <Link href="https://github.com/jpsahoo14/apollo-inspector-ui" inline rel="noopener noreferrer"> Read Me </Link>
```
Comment (jpsahoo14) [inline src/operations-tracker-header/operations-tracker-header.tsx:60]: This should be the link

https://github.com/jpsahoo14/apollo-inspector-ui/blob/main/extension/readme.md

File: src/verbose-operation/data-grid-view.tsx [hunk 2]
```diff
@@ -129,11 +152,16 @@ export const DataGridView = React.memo((props: IDataGridView) => {
...
   classes: IClasses,
   handleToggleFilters: () => void,
-  operationsState: IOperationsReducerState
+  operationsState: IOperationsReducerState,
+  debouncedFilter: DebouncedFunc<(e: React.SyntheticEvent) => void>
 ) => (
+
   <div className={classes.headers}>
+    <div className={classes.searchBar}>
+      <Search onSearchChange={debouncedFilter} />
```
Comment (jpsahoo14) [inline src/verbose-operation/data-grid-view.tsx:161]: Test if search is working fone?

**Comment by jpsahoo14** [comment]
Add image in description

File: src/operations-tracker-header/operations-tracker-header.tsx (hunk 1 above)
Comment (VyshnaviDasari) [reply src/operations-tracker-header/operations-tracker-header.tsx:60]: Changes

File: src/verbose-operation/data-grid-view.tsx (hunk 2 above)
Comment (VyshnaviDasari) [reply src/verbose-operation/data-grid-view.tsx:161]: Tested

File: src/operations-tracker-header/operations-tracker-header.tsx (hunk 1 above)
Comment (jpsahoo14) [reply src/operations-tracker-header/operations-tracker-header.tsx:60]: I still see the old url

**Review by jpsahoo14 — APPROVED** [review]

**Pull request merged**